scripts/prepare-release.sh

```diff
@@ -45,133 +45,77 @@ else
+  yq e -i ".metadata.name=\"managed-api-service.v$VERSION\"" config/manifests-$OPERATOR_TYPE/bases/managed-api-service.clusterserviceversion.yaml
+  yq e -i ".spec.version=\"$VERSION\"" config/manifests-$OPERATOR_TYPE/bases/managed-api-service.clusterserviceversion.yaml
   if [[ "${VERSION}" != "${PREVIOUS_VERSION}" ]]; then
-    yq e -i ".spec.replaces=\"$OLM_TYPE.v$PREVIOUS_VERSION\"" config/manifests-$OPERATOR_TYPE/bases/$OLM_TYPE.clusterserviceversion.yaml
+    yq e -i ".spec.replaces=\"managed-api-service.v$PREVIOUS_VERSION\"" config/manifests-$OPERATOR_TYPE/bases/managed-api-service.clusterserviceversion.yaml
   fi
 }
 
+create_or_update_csv() {
+  "${KUSTOMIZE[@]}" build config/manifests-$OPERATOR_TYPE | operator-sdk generate bundle --kustomize-dir config/manifests-$OPERATOR_TYPE --output-dir bundles/$OLM_TYPE/$VERSION --version $VERSION --default-channel stable --package ${PACKAGE_NAME} --channels stable
+}
+
 set_version() {
   if [[ -z "$PREVIOUS_VERSION" ]]
     then
       echo "No previous version please set correct values in the Makefile and version/version.go files"
     else
       case $OLM_TYPE in
-        "integreatly-operator")
-          "${SED_INLINE[@]}" -E "s/RHMI_TAG\s+\?=\s+$PREVIOUS_VERSION/RHMI_TAG \?= $VERSION/g" Makefile
-          "${SED_INLINE[@]}" -E "s/version\s+=\s+\"$PREVIOUS_VERSION\"/version = \"$VERSION\"/g" version/version.go
-          yq e -i ".channels[0].currentCSV=\"$OLM_TYPE.v$VERSION\"" bundles/$OLM_TYPE/*.package.yaml
-          ;;
         "managed-api-service")
           "${SED_INLINE[@]}" -E "s/RHOAM_TAG\s+\?=\s+$PREVIOUS_VERSION/RHOAM_TAG \?= $VERSION/g" Makefile
           "${SED_INLINE[@]}" -E "s/managedAPIVersion\s+=\s+\"$PREVIOUS_VERSION\"/managedAPIVersion = \"$VERSION\"/g" version/version.go
           yq e -i ".channels[0].currentCSV=\"$OLM_TYPE.v$VERSION\"" bundles/$OLM_TYPE/*.package.yaml
           ;;
-        *)
-          echo "No version found for install type : $(OLM_TYPE)"
-          ;;
-      esac
-  fi
-}
-
-set_installation_type() {
-  if [[ -z "$PREVIOUS_VERSION" ]]
-    then
-      echo "No previous version please set correct values in the Makefile and version/version.go files"
-    else
-      case $OLM_TYPE in
-        "integreatly-operator")
-          echo "using default INSTALLATION_TYPE found in deploy/operator.yaml"
-          ;;
-        "managed-api-service")
-          yq e -i '(.spec.install.spec.deployments[0].spec.template.spec.containers[0].env.[] | select(.name=="INSTALLATION_TYPE") | .value) = "managed-api" ' bundles/$OLM_TYPE/${VERSION}/manifests/$OLM_TYPE.clusterserviceversion.yaml
+        "multitenant-managed-api-service")
+          "${SED_INLINE[@]}" -E "s/MULTITENANT_RHOAM_TAG\s+\?=\s+$PREVIOUS_VERSION/MULTITENANT_RHOAM_TAG \?= $VERSION/g" Makefile
+          "${SED_INLINE[@]}" -E "s/multitenantManagedAPIVersion\s+=\s+\"$PREVIOUS_VERSION\"/multitenantManagedAPIVersion = \"$VERSION\"/g" version/version.go
```

Comment:
@carlkyrillos I don't think we need this, see comment below

Reply:
Addressed in latest commit